packages/react/toggle-group/src/ToggleGroup.tsx

```diff
@@ -275,8 +278,12 @@ const ToggleGroupItemImpl = React.forwardRef<ToggleGroupItemImplElement, ToggleG
   (props: ScopedProps<ToggleGroupItemImplProps>, forwardedRef) => {
     const { __scopeToggleGroup, value, ...itemProps } = props;
     const valueContext = useToggleGroupValueContext(ITEM_NAME, __scopeToggleGroup);
+    const pressed = valueContext.value.includes(value);
```

Comment:
Just curious about declaring `pressed` again vs taking from `props`. Is that to keep the prims props distinct from the underlying `Toggle`?

Reply:
Ah yeah good catch, it seems at this point it will be a prop that will have been passed down already internally right?

Reply:
good catch indeed 👍